src/main/telemetry/telemetry.h

```diff
@@ -42,7 +42,8 @@ typedef enum {
 
 typedef enum {
     FRSKY_UNIT_METRICS = 0,
-    FRSKY_UNIT_IMPERIALS
+    FRSKY_UNIT_IMPERIALS,
+    FRSKY_UNIT_BRITISH
```

Comment:
Keeping this enum around, and having to keep it in-synch with `osd_unit_e` is just making this more brittle and hard to maintain than it need be. I'd combine the two into a `unit_e` enum and use this in both places.

Reply:
Would https://github.com/betaflight/betaflight/blob/master/src/main/common/utils.h be the right place for the enum?
Then units in configurator should move from osd to configuration tab? Also the two current unit vars could be reduced to one in this way.

Reply:
About Configurator, not needed to move. The enum place is indifferent for this, what is interesting for Configurator is how many variables/msp use it.